src/Verifier/verifier_test.go

```diff
@@ -34,7 +34,7 @@ func TestVerifierFunctionDuplicateDeletion(t *testing.T) {
 	createCsvFiles(csvfile1)
 	defer cleanUp()
 
-	Verifier("./testFile1.csv", "./testFile2.csv")
+	DuplicateRemoval("./testFile1.csv", "./testFile2.csv")
 
 	csv1, err := os.ReadFile("./testFile1.csv")
 	if err != nil {
```

Comment:
I think verifier also needs a stand-alone test, unless I've missed it:)

Reply:
This has now been fixed